offchain/inspect-server/src/server.rs

```diff
@@ -18,7 +18,8 @@ pub fn create(
     config: &InspectServerConfig,
     inspect_client: InspectClient,
 ) -> std::io::Result<Server> {
-    let inspect_path = config.inspect_path_prefix.clone() + "/{payload:.*}";
+    let inspect_base_path = config.inspect_path_prefix.clone();
```

Comment:
I think that this path doesn't need to be configurable anymore. Let's remove this configuration and simplify the code.

Reply:
Not sure about that, @gligneul. I have talked to @endersonmaia and having a configurable inspect path should be useful for load balancing purposes or something like that..

Reply:
@marcelstanley I had a conversation with @gligneul , and checking the current AWS ALB configuration, I think we can live with a hard-coded `/inspect` path.

I shall test this at AWS and give some feedback here if I find some problems.